### DISCUSSION TITLE
Request Approved but not received benefits

### DISCUSSION THREAD

**Discussion by bytechconsultants** [community/community#132026] [GitHub Education]
### Hi everyone,

My Request has approved on 11th July but the benefits of Github Education is not still available to me.Please give the benefits of Github student developer pack as soon as possible.

**Comment by naveenalla3000** [comment]
you may receive 2-4 days after its approved.
mark as answer if it is helpful.

**Comment by Ali01129** [comment]
if it is approved just go to copilot and select a plan and then they will tell you that you have student account and then it will be free to use.

**Comment by rathneshh** [comment]
Same I got approved as of 10-07-2024, but I still haven't received my benefits. Hoping to get them soon.
